packages/account/src/Components/terms-of-use/terms-of-use.scss

```diff
@@ -4,6 +4,7 @@
     flex-grow: 1;
     margin: 0 8rem !important;
     width: 84% !important;
+    padding-bottom: unset;
```

Comment:
since we set the `padding-bottom` at the top level, isn't it available on mobile as well as there's no override?
Am I missing something?